src/dotnet-uninstall/MacOs/ListCommandExec.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.CommandLine.Rendering.Views;
+using System.Linq;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Commands;
+
+namespace Microsoft.DotNet.Tools.Uninstall.MacOs
+{
+    internal static class ListCommandExec
+    {
+        public static readonly IEnumerable<BundleTypePrintInfo> SupportedBundleTypes =
+            new BundleTypePrintInfo[]
+            {
+                new BundleTypePrintInfo<SdkVersion>(LocalizableStrings.ListCommandSdkHeader),
+                new BundleTypePrintInfo<RuntimeVersion>(LocalizableStrings.ListCommandRuntimeHeader)
+            };
+
+        public static GridView GetGridView(IList<Bundle> bundles)
+        {
+            var gridView = new GridView();
+
+            gridView.SetColumns(Enumerable.Repeat(ColumnDefinition.SizeToContent(), 3).ToArray());
+            gridView.SetRows(Enumerable.Repeat(RowDefinition.SizeToContent(), Math.Max(bundles.Count, 1)).ToArray());
+
+            foreach (var (bundle, index) in bundles.Select((bundle, index) => (bundle, index)))
+            {
+                gridView.SetChild(new ContentView(string.Empty), 0, index);
+                gridView.SetChild(new ContentView(bundle.Version.ToString()), 1, index);
+                gridView.SetChild(new ContentView($"({bundle.Arch.ToString().ToLower()})"), 2, index);
```

Comment:
For macOS, this will always output `x64`, right?  Is that useful information?  It appears that this line is the only different between this implementation and the one for Windows.

If we want to keep this, perhaps we should set the display name for the bundle to `BundleArch.X64.ToString().ToLowerInvariant()` (or simply just "x64") when creating them in `FileSystemExplorer.GetInstalledBundles` and thus could potentially unify the `ListCommandExec` implementations (i.e. always output `bundle.DisplayName`) unless we foresee different output based on platform for the future?

Reply:
Yes, you're right - for macOS this will always output `x64`. However, I personally think it will still be useful information because we have `--x64` and `--x86` options. Users may get confused when looking at the help page if we don't have `x64` in the list output.

I'm not sure if putting `x64` in the display name is a good idea because `x64` is an architecture type but not a display name. We don't have a display name for macOS installations. If we put `x64` in `DisplayName`, it will be confusing for other developers of this tool. We could instead "fake" a display name such as "Microsoft .NET Core 2.2.300 (x64)` in place of the architecture type.

Reply:
A derived display name might give a more consistent UX between platforms. @KathleenDollard what's your take?

Reply:
I have indeed asked @KathleenDollard about this, and the answer was it's okay to have UI different between platforms. That being said, we could definitely tweak this more for a better UX.

Reply:
I don't find "x64" repeated ad infinitum to be useful information, but if this has already been discussed with Kathleen, I won't hold up this PR.